### DISCUSSION TITLE
How can /api/auth be secured?

### DISCUSSION THREAD

**Discussion by rikyreas14** [rabbitmq/rabbitmq-server#10948] [Other]
### Describe the bug

If I integrate rabbitmq with azure /api/auth is exposing sensitive data through non-secured apis

### Reproduction steps

1. enable management plugin
2. enable azure oauth
3.access /api/auth
4.
...

### Expected behavior

is there any workaround or we can add security to api/auth

**Comment by michaelklishin** [comment]
We cannot suggest much since you haven't explained what you consider sensitive or why.
"Enable Azure OAuth" is not a reproduction step.

We do not guess in this community.

OAuth 2 and OpenID Connect generally have well known API endpoints used for discovery and and different workflows where different parties (tools, libraries) involved may or may not have a way to authenticate with each other using an "out of band" mechanism.

If you want to protect a certain API endpoint, you can put a proxy in front of it and restrict network access to the endpoint that way.
